solutions/usaco-1064.mdx

```diff
@@ -108,4 +108,111 @@ int main() {
 ```
 
 </CPPSection>
+	
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class StuckInARut {
+	static int[] xCoordinates;
+	static int[] yCoordinates;
```

Comment:
you could just change this to static int[] x and static int[] y to save space later on (make sure to add a brief explanation of what they represent if you do)

Reply:
i don't think i need to do this anymore because I'm under the limit

Reply:
well it would let you put the if statements all in one line, which would be more readable